### PR TITLE
docker_host_info and docker_node: fix return variable names

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Follow-up PR to #54124.

Changes the name of one of the return values of `docker_host_info` from `host_facts` to `host_info`.

Also changes the name of `docker_node`'s return value `node_facts` to `node`, so that it is more similar to other module's return values and also fits `docker_node_info`'s `nodes` return value name.

##### ISSUE TYPE
- Bugfix Pull Request
- Docs Pull Request

##### COMPONENT NAME
docker_host_info
docker_node

### CODE REVIEW

File: lib/ansible/modules/cloud/docker/docker_node.py [hunk 1]
```diff
@@ -122,7 +122,7 @@
 '''
 
 RETURN = '''
-node_facts:
+node:
```
Comment (WojciechowskiPiotr) [inline lib/ansible/modules/cloud/docker/docker_node.py:125]: Why not `node_info`?

Comment (felixfontein) [reply lib/ansible/modules/cloud/docker/docker_node.py:125]: For consistency: `docker_node_facts` returns `nodes`, and most other modules which operate on one object return a variable whose name is the object type in singular without some suffix.

**Comment by felixfontein** [comment]
I'm also open for other suggestions (also for other return values which are new for 2.8). They should be consistent, expressive enough and reasonable short, though... ([overview of current return variables](https://github.com/ansible/ansible/pull/54124#issuecomment-475036128))

**Comment by WojciechowskiPiotr** [comment]
Most values does not contain the `_facts` so it is easier to remove from remaining

**Comment by felixfontein** [comment]
We could also remove it from docker_swarm_info, but we can't remove it from docker_swarm. I'm tending to leave it there for docker_swarm_info as well, so that both docker_swarm and docker_swarm_info behave similarly. What do you think?

**Comment by WojciechowskiPiotr** [comment]
Lets leave consistent between those two

**Comment by WojciechowskiPiotr** [comment]
shipit

**Comment by felixfontein** [comment]
shipit

**Comment by felixfontein** [comment]
@WojciechowskiPiotr thanks for reviewing!

**Pull request merged**